src/logion/controllers/locrequest.controller.ts

```diff
@@ -263,7 +263,7 @@ export class LocRequestController extends ApiController {
         const description = request.getDescription();
         if (description.requesterAddress) {
             const protections = await this.protectionRequestRepository.findBy(new FetchProtectionRequestsSpecification({
-                expectedStatuses: [ "ACCEPTED", "ACTIVATED" ],
+                expectedStatuses: [ "ACCEPTED", "ACTIVATED", "ACCEPTED_CANCELLED" ],
```

Comment:
If the request was cancelled, is it desirable that user identity is found?

Reply:
Yes. The protection request is cancelled, but the associated identity LOC was created and is still valid.

Reply:
OK